packages/node_modules/@node-red/nodes/core/core/58-debug.html

```diff
@@ -246,7 +247,8 @@
             RED.sidebar.removeTab("debug");
             RED.events.off("workspace:change", this.refreshMessageList);
             window.removeEventListener("message",this.handleWindowMessage);
-            RED.actions.remove("core:show-debug");
+            RED.actions.remove("core:show-debug-tab");
```

Comment:
I couldn't find any other reference to `core:show-debug` so sneaked this change in to this PR.

```
$ grep -r core:show-debug *
packages/node_modules/@node-red/nodes/core/core/58-debug.html:                action: "core:show-debug-tab"
packages/node_modules/@node-red/nodes/core/core/58-debug.html:            RED.actions.add("core:show-debug-tab",function() { RED.sidebar.show('debug'); });
packages/node_modules/@node-red/nodes/core/core/58-debug.html:            RED.actions.remove("core:show-debug-tab");
packages/node_modules/@node-red/editor-client/locales/ko/infotips.json:        "tip11": "{{core:show-info-tab}}를 사용하여 정보탭을 표시하거나 {{core:show-debug-tab}}를 사용하여 디버그탭을 표시할  수 있습니다.",
packages/node_modules/@node-red/editor-client/locales/zh-CN/infotips.json:        "tip11": "{{core:show-info-tab}} 可以显示「信息」标签页。 {{core:show-debug-tab}} 可以显示「调试」标签页。",
packages/node_modules/@node-red/editor-client/locales/ja/infotips.json:        "tip11": "{{core:show-info-tab}} で「情報」タブを表示します。 {{core:show-debug-tab}} で「デバッグ」タブを表示します。",
packages/node_modules/@node-red/editor-client/locales/en-US/infotips.json:        "tip11" : "Show the Info tab with {{core:show-info-tab}} or the Debug tab with {{core:show-debug-tab}}",
packages/node_modules/@node-red/editor-client/locales/de/infotips.json:    "tip11" : "Die Registerkarte \"Info\" mit {{ core:show-info-tab }} oder der Registerkarte \"Debug\" mit {{ core:show-debug-tab }} anzeigen",
packages/node_modules/@node-red/editor-client/src/js/keymap.json:        "ctrl-g d": "core:show-debug-tab",
packages/node_modules/@node-red/editor-client/public/red/keymap.json:        "ctrl-g d": "core:show-debug-tab",
```